dashboard/config/libraries/Outbreak.interpreted.js

```diff
@@ -0,0 +1,158 @@
+/* -------------- START OF MONSTER BEHAVIOR DEFINITIONS -------------- */
+// Owned By Curriculum
+function congregatingBehavior(spriteIdArg) {
+  var speed = getProp(spriteIdArg, "speed") || 1;
+  var targetX = 200 + randomNumber(-20, 20);
+  var targetY = 200 + randomNumber(-20, 20);
+  moveToward(spriteIdArg, 1, {x: targetX, y: targetY});
```

Comment:
what is the second arg (1) here? a comment would help. in the command definition, it looks like it's `distance`?

Reply:
ah, it looks like this is how we're using `speed` elsewhere